app/views/mailer/alert_project_creator_about_new_member.erb

```diff
@@ -0,0 +1,6 @@
+Hi <%= @project_creator.display_name %>,
+====================================================================================
+
+<%= @user.email %> just joined your project <%= @project.title %>, you can reach out and personally welcome them.
+
+You are receiving this email because you registered at http://www.agileventures.org. If you wish to be removed from this mailing list, adjust your profile settings as http://www.agileventures.org.
```

Comment:
Please login at av.org and go to your profile page and edit "Receive site emails" to be unchecked - if you are already logged in go directly to: http://www.agileventures.org/user/edit